### PR TITLE
[ci skip] Fix UseCooldownComponent javadocs

### PR DESCRIPTION
**Pull request by Leguan16**

### CODE REVIEW

File: patches/api/0053-Fix-upstream-javadocs.patch [hunk 1]
```diff
@@ -1683,6 +1683,52 @@ index 43f0df04f3cdff7d7db73321a2886f3a737e3c9f..5c741228b2338a7c4de2fe736eb78951
...
+- * Represents a component which determines the cooldown applied to use of this
+- * item.
++ * Represents a component which determines the cooldown applied to use this item.
+  */
+ @ApiStatus.Experimental
+ public interface UseCooldownComponent extends ConfigurationSerializable {
+ 
+     /**
+-     * Gets the time in seconds it will take for this item to be eaten.
++     * Gets the time in seconds it will take for this item to be used again.
```
Comment (Machine-Maker) [inline patches/api/0053-Fix-upstream-javadocs.patch]: `for this item to be available for use again`

File: patches/api/0053-Fix-upstream-javadocs.patch [hunk 2]
```diff
@@ -1683,6 +1683,52 @@ index 43f0df04f3cdff7d7db73321a2886f3a737e3c9f..5c741228b2338a7c4de2fe736eb78951
...
+      *
+-     * @return eat time
++     * @return cooldown time
+      */
+     float getCooldownSeconds();
+ 
+     /**
+-     * Sets the time in seconds it will take for this item to be eaten.
++     * Sets the time in seconds it will take for this item to be used again.
```
Comment (Machine-Maker) [inline patches/api/0053-Fix-upstream-javadocs.patch]: `for this item to be available for use again`

**Review by Machine-Maker — CHANGES_REQUESTED** [review]

**Review by Machine-Maker — APPROVED** [review]

File: patches/api/0053-Fix-upstream-javadocs.patch [hunk 3]
```diff
@@ -1683,6 +1683,52 @@ index 43f0df04f3cdff7d7db73321a2886f3a737e3c9f..5c741228b2338a7c4de2fe736eb78951
...
+index 04b4b08021a841542b3d5e6f3f1c465af3899281..6a0d551df9b4fbc3ef2b6140b93f2019e56faf69 100644
+--- a/src/main/java/org/bukkit/inventory/meta/components/UseCooldownComponent.java
++++ b/src/main/java/org/bukkit/inventory/meta/components/UseCooldownComponent.java
+@@ -6,25 +6,24 @@ import org.jetbrains.annotations.ApiStatus;
+ import org.jetbrains.annotations.Nullable;
+ 
+ /**
+- * Represents a component which determines the cooldown applied to use of this
+- * item.
++ * Represents a component which determines the cooldown applied to use this item.
```
Comment (MiniDigger) [inline patches/api/0053-Fix-upstream-javadocs.patch]:
```suggestion
+ * Represents a component which determines the cooldown applied when using this item before it is available for use again.
```

Comment (Leguan16) [reply patches/api/0053-Fix-upstream-javadocs.patch]: changed it.

**Review by MiniDigger — APPROVED** [review]

**Review by lynxplay — APPROVED** [review]

**Pull request merged**